src/views/Nft/market/Collection/IndividualNFTPage/index.tsx

```diff
@@ -49,9 +54,38 @@ const IndividualNFTPage = () => {
     fetchTokens()
   }, [collectionAddress, setAttributesDistribution])
 
-  if (allBunnies.length === 0 || !cheapestBunny) {
-    // TODO redirect to nft market page if collection or bunny id does not exist
-    return null
+  useEffect(() => {
```

Comment:
in case there are no bunnies on sale - show generic bunny data

Note: the condition how we determine "no bunnies on sale" will change when I'll refactor the subgraph query to take into account possibility of 10k Dollops on the market. But for now it just looks in redux state and if not - show this